### ISSUE TITLE
`authenticatedRoute` not working on Ember >= 3.6

### ISSUE THREAD

**Issue by kategengler** [Vestorly/torii#431]
On Ember >= 3.6, authenticatedRoute does not hit the `accessDenied` action. It works in Ember 3.5

I've created a minimal reproduction here: https://github.com/kategengler/torii-test

**Comment by veelenga** [comment]
@kategengler is this still an issue on Ember 3.8?

**Comment by pollingj** [comment]
@veelenga yes, I'm still seeing it.  So @kategengler's fix has been very helpful.
